extension/src/utilities.rs

```diff
@@ -0,0 +1,63 @@
+use pgx::*;
+use pg_sys::{TimestampTz};
+
+#[pg_extern(schema = "timescale_analytics_experimental")]
+pub fn generate_periodic_normal_series(
```

Comment:
we will probably want some tests for this at some point, but for now, I think it's fine to be a bit vague as it's only a utility for internal use for now...